tests/test_georaster.py

```diff
@@ -1247,7 +1271,7 @@ def test_interp(self) -> None:
             list_z_ind.append(z_ind)
 
         # First order interpolation
-        rpts = r.interp_points(pts, order=1, area_or_point="Area")
+        rpts = r.interp_points(pts, order=1, shift_area_or_point=True)
```

Comment:
Why is `shift_area_or_point` set to True here? If it was "Area" in the previous implementation, it should not be shifted, right?

Reply:
Yes, not needed! Removed.